From 78486df16c33b1b339d891380a97c577768a4e6f Mon Sep 17 00:00:00 2001
From: Fakeman <55951275+fakemanoan@users.noreply.github.com>
Date: Fri, 21 Jun 2024 09:40:34 +0100
Subject: [PATCH] Allow build warnings from /hardware

SLSI BSP has warnings currently despite building correctly, so allow them.

Change-Id: I4d06f4a947079467f695dca0071cf56943c60ab5
---
 cc/config/global.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cc/config/global.go b/cc/config/global.go
index 473c806..b03641d 100644
--- a/cc/config/global.go
+++ b/cc/config/global.go
@@ -135,6 +135,7 @@ var (
 	WarningAllowedProjects = []string{
 		"device/",
 		"vendor/",
+		"hardware/",
 	}
 
 	// Directories with warnings from Android.mk files.
-- 
2.25.1

